internal/resource/route.go

```diff
@@ -126,13 +127,19 @@ func (r Route) ProcessDefaults(ctx context.Context) error {
 	if r.Route == nil {
 		return fmt.Errorf("invalid nil resource")
 	}
-	if len(r.Route.Protocols) == 0 {
-		err := mergo.Merge(r.Route, defaultRoute,
-			mergo.WithTransformers(wrappersPBTransformer{}))
-		if err != nil {
-			return err
+	// When protocols has 'grpc' or 'grpcs', 'strip_path' cannot be set.
+	// If 'strip_path' is null, the default value (true) would be applied,
+	// so we set it to 'false' in order to prevent that.
+	if lo.Contains(r.Route.Protocols, "grpc") || lo.Contains(r.Route.Protocols, "grpcs") {
```

Comment:
There's a better way to do that w/ `lo` 😀
```suggestion
	if lo.Some(r.Route.Protocols, []string{typedefs.ProtocolGRPC, typedefs.ProtocolGRPCS}) {
```

Reply:
oh, nice!